CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 2.8...3.19)
```

Comment:
Why do you set that 3.19 is the maximum? What will happen in future? Have you tried the 3.20? What are the problems of the current latest release?

Reply:
It does not work like that. It basically says that we require the CMake to be *at least* somewhere between 2.8...3.19. There is no cap on the version, it is still *minimum required*.

Reply:
Yeah, and 3.19 is what I have ATM and can confirm that it is working.